src/styles/style.css

```diff
@@ -0,0 +1,20 @@
+@import url(Blocks/card.css);
+@import url(Blocks/stars.css);
+
+.page {
+  font-family: Roboto, Helvetica, sans-serif;
+
+  --ColorSmallText: #616070;
+  --ColorBlue: #00acdc;
+  --ColorWhite: #fff;
+  --ColorBigText: #060b35;

```

Comment:
variables should be in `:root` pseudo-selector

Reply:
I had some problems with the deploy. This problem was fixed and all recommendations were made.